test/OpenTelemetry.Tests/Trace/SamplersTest.cs

```diff
@@ -252,5 +252,29 @@ public void SamplersDoesNotImpactTraceStateWhenUsingNull(SamplingDecision sampli
                 Assert.Equal(parentTraceState, activity.TraceStateString);
             }
         }
+
+        [Fact]
+        public void SamplerExceptionBubblesUpTest()
+        {
+            // TODO: Discuss: An exception thrown in custom sampler probably
+            // should not blow up like this?
```

Comment:
Remove TODO assuming we agree this is the behavior we want.

Reply:
or leave a comment saying this is by design, and link to this PR/issues, so a future reader can easily find all the prior discussions on this topic.

Reply:
Updated